util/coin/fixtures.go

```diff
@@ -0,0 +1,48 @@
+package coin
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	appparams "github.com/umee-network/umee/v4/app/params"
+	leveragetypes "github.com/umee-network/umee/v4/x/leverage/types"
+)
+
+const umee = appparams.BondDenom
+
+// common coins used in tests
+var (
+	UumeeDenom = leveragetypes.ToUTokenDenom(umee)
+	Umee1      = New(umee, 1)
+	Umee10k    = New(umee, 10_000)
+	UUmee1     = Utoken(umee, 1)
+
+	Umee1dec = DecF(umee, 1)
```

Comment:
```suggestion
	// the uToken denom "u/uumee"
	UumeeDenom = leveragetypes.ToUTokenDenom(umee)
	// a Coin containing 1uumee
	Umee1      = New(umee, 1)
	// a Coin containing 10000uumee
	Umee10k    = New(umee, 10_000)
	// a Coin containing 1 u/uumee
	UUmee1     = Utoken(umee, 1)

	// a DecCoin containing 1 uumee
	Umee1dec = DecF(umee, 1)
```
This might help in the (rare) case where someone needs to mouse over one of these fixtures to examine the specifics of a test case. Do you think it's worth commenting?

Reply:
Sounds good.